config/webpack.config.js

```diff
@@ -106,6 +106,14 @@ const configGenerator = (options) => {
         {
           test: /\.json$/,
           loader: 'json-loader'
+        },
+        {
+          test: /react-jsonschema-form\/lib\/components\/(widgets|fields\/ObjectField|fields\/ArrayField)/,
```

Comment:
I'm a little confused what this is doing with `fields/ObjectField` and `fields/ArrayField`. I assume we want to not include those because we have our own versions? Is this excluding them? Why are they not in the `exclude` array below?